source/about/cloud-subscriptions.rst

```diff
@@ -4,27 +4,83 @@ Cloud subscriptions
 Purchase a Cloud subscription
 ------------------------------
 
-Mattermost Cloud subscriptions start with a 14-day trial. When you're near the end of your trial, a banner within Mattermost will notify you to upgrade to a subscription plan.
+Mattermost Cloud can be used free forever with Mattermost Starter.
+
+You can also start a free, 30-day trial of Mattermost Enterprise to explore additional features. When you're near the end of your trial, Mattermost will prompt you to upgrade to a subscription plan. If you'd like to continue with Mattermost Enterprise, please contact our `Sales team <mailto:sales@mattermost.com>`_. 
+
+If, after 30 days, you choose not to continue using Mattermost Enterprise, your subscription will revert back to Mattermost Starter. 
+
+Alternatively, you can upgrade to a monthly per-user subscription, with Mattermost Professional.
 
 1. Select **Subscribe now** from the banner.
-2. Provide your credit card details. 
+2. Provide your credit card details.
+  - Your credit card will be verified and you'll be charged at the end of the billing period. If you'd prefer an annual Cloud subscription, please contact our Sales team for a quote.
 3. Provide your billing address.
 4. (Optional) Select a subscription plan.
 5. Select **Subscribe**.
 
-Your credit card will be verified and you'll be charged at the end of the billing period. If you'd prefer an annual Cloud subscription, please contact our Sales team for a quote.
-
 View subscription details
 -------------------------
 
-Go to **System Console > Billing & Account > Subscription** to access the following subscription details:
+The subscription screen outlines the details of your Mattermost Cloud account, including the cost per user per month and the number of users currently registered in your workspace. You can also find a summary of your last invoice in the same page, and any `workspace limits <https://docs.mattermost.com/mattermost-limits.html>`_ set for your subscription.
+
+Billing period
+--------------
+
+After you begin your Mattermost Cloud paid subscription, your first charge is applied at the end of the calendar month. Subsequent billing periods begin at 12 AM UTC on the first day of each calendar month and end at 11:59 PM UTC on the final day of the same calendar month. Per-user pricing for each billing period will be based on the number of registered users.
+
+Invoices appear in the Customer Portal immediately after the billing period ends.
+
+Billing periods begin at 12 AM UTC on the first day of each calendar month and end at 11:59 PM UTC on the final day of the same calendar month, at which time subscription costs are calculated, and a credit card charge is automatically attempted. It may take up to a day for our billing systems to calculate, charge your credit card, and issue your invoice. Once charges are completed, the invoice will show as *Paid*. The invoice is a downloadable PDF file.
```

Comment:
```suggestion
It may take up to a day for our billing systems to calculate, charge your credit card, and issue your invoice. Once charges are completed, the invoice will show as *Paid*. The invoice is a downloadable PDF file.
```